src/Models/Message.php

```diff
@@ -84,12 +85,13 @@ public function conversation()
      *
      * @return Model
      */
-    public function send(Conversation $conversation, string $body, Participation $participant, string $type = 'text'): Model
+    public function send(Conversation $conversation, string $body, Participation $participant, string $type = 'text', string $data = ''): Model
```

Comment:
Can we make `$data` an array just like we have with conversation? https://github.com/musonza/chat/blob/851e9199e7131bdf834c45c931ce5896bee4658f/src/Models/Conversation.php#L26

Reply:
Yes, it would be great. I will change it and push it soon.